CMakeLists.txt

```diff
@@ -268,3 +269,8 @@ IF(BUILD_EXAMPLES)
   MESSAGE(STATUS "Configurating examples")
   ADD_SUBDIRECTORY(${MY_DIR}/examples)
 ENDIF()
+
+IF(BUILD_OPENNI2FREENECT2DRIVER)
+  MESSAGE(STATUS "Configurating OpenNI2 Freenect2 Driver")
```

Comment:
"Configurating" was a typo. Should use "Configuring". You can fix the typo in the above "Configurating examples" too.